client/pages/sign-up.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <AuthForm v-if="!accountCreated" v-slot="{ invalid }" @submit="register">
+    <AuthAlert :errors="errors">
+      Error creating account, please fix the following errors:
+    </AuthAlert>
+    <InputField
+      v-for="field in fields"
+      :id="field.id"
+      :key="field.index"
+      :field-name="field.name"
+      :field-type="field.type"
+      :field-options="field.options"
+      :rules="field.rules"
+      :inputvalue.sync="field.value"
+    />
+    <AuthSubmit :disabled="invalid">Submit</AuthSubmit>
+    <p class="self-center mt-3">
+      Already have an account?
+      <NuxtLink to="/login" class="text-blue">Click here to log in</NuxtLink>
+    </p>
+  </AuthForm>
+  <AuthForm
+    v-else
+    class="place-items-center gap-5 font-bold text-2xl text-white"
+  >
+    <h1 class="text-3xl">Congratulations {{ name }}!</h1>
+    <p>
+      Your new Elucidate account<wbr />
+      has been created
+    </p>
+    <font-awesome-icon
+      :icon="['fas', 'fa-face-smile-beam']"
+      class="text-[10rem] my-10"
+    />
+    <p>
+      Please proceed to the
+      <NuxtLink to="/quiz" class="text-blue">Home Page</NuxtLink><wbr /> or
+      <NuxtLink to="/login" class="text-blue">Search</NuxtLink>
+      for quizzes
+    </p>
+  </AuthForm>
+</template>
+
+<script>
+let count = 0;
+export default {
+  name: 'SignupPage',
+  auth: false,
+  layout: 'auth',
+  data: () => ({
+    title: 'Sign-Up',
+    accountCreated: false,
+    name: '',
```

Comment:
Is this field used?

Reply:
Yes, on line 29 (`<h1 class="text-3xl">Congratulations {{ name }}!</h1>`)

Reply:
I suggest yeeting it in preference to redirecting to the login screen on successful sign-up